app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts_counter + 1)
+  end
```

Comment:
- [ ] It would be nice if we use one of [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), the `after_save` method to call the private method `update_posts_counter`. Let use kindly make the method private